### PR TITLE
Add thermostat

### PR DESCRIPTION
**Pull request by dhylands**

### CODE REVIEW

**Comment by mrstegeman** [comment]
Lots of eslint errors...

**Comment by dhylands** [comment]
Whoops - lint errors should be fixed now.

**Comment by mrstegeman** [comment]
Now there are unit test failures.

**Comment by dhylands** [comment]
I knew I had to change that file...

File: zwave-classifier.js [hunk 1]
```diff
@@ -786,6 +813,119 @@ class ZWaveClassifier {
...
+    const zwValue = node.zwValues[fanStateValueId];
+    if (!zwValue) {
+      return;
+    }
+    this.addProperty(
+      node,
+      'fanState',
+      {
+        title: 'Fan State',
+        type: 'string',
```
Comment (mrstegeman) [inline zwave-classifier.js]: Seems like this one should be readOnly.

File: zwave-classifier.js [hunk 2]
```diff
@@ -1693,18 +1833,52 @@ class ZWaveClassifier {
...
+      // be an enumeration.
+      node.doorLockState = this.addProperty(
         node,
         'locked',
+        {
+          '@type': 'LockedProperty',
+          type: 'string',
+          title: 'State',
+          enum: ['locked', 'unlocked', 'jammed', 'unknown'],
+          readonly: true,
```
Comment (mrstegeman) [inline zwave-classifier.js]: s/readonly/readOnly/

**Pull request merged**